unmaintained/see_dat/see_dat.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"flag"
+	master_ui "github.com/chrislusf/seaweedfs/weed/server/volume_server_ui"
```

Comment:
better not use something for ui for this code, adding unnecessary dependency.

Reply:
refactored to util.BytesToHumanReadable